CONTRIBUTING.rst

```diff
@@ -24,7 +24,7 @@ Coding Guidelines
 =================
 
 1.  With the `rocBLAS device memory allocation
-    system <https://github.com/ROCmSoftwarePlatform/rocBLAS/blob/develop/docs/Device_Memory_Allocation.pdf>`__,
+    system <https://github.com/ROCm/rocBLAS/blob/release/rocm-rel-6.2/docs/Device_Memory_Allocation.pdf>`__,
```

Comment:
I think that file was deleted by @rkamd so rather than fix the link to the outdated document , maybe replace with hyperlink to new section?

Reply:
The file was deleted in ROCm 6.3, it is still there in ROCm 6.2, but the link is broken in 6.2 because it points to develop rather than to release/rocm-rel-6.2

Reply:
Yes but the content I think was stale even in 6.2   :-)    See what Ruban says